tests/test_project.py

```diff
@@ -8,3 +10,27 @@ def test_project_folder(cookies):
     assert project.exception is None
     assert project.project.basename == 'my-python-project'
     assert project.project.isdir()
+
+
+def run(command: str, dirpath: os.PathLike) -> subprocess.CompletedProcess:
+    return subprocess.run(shlex.split(command),
+                          stdout=subprocess.PIPE,
+                          stderr=subprocess.PIPE,
+                          cwd=dirpath,
+                          text=True)
+
+
+def test_pytest(cookies):
+    result = cookies.bake()
+    env_output = run('python3 -m venv env', result.project)
+    assert env_output.returncode == 0
+    latest_pip_output = run('env/bin/pip3 install --upgrade pip setuptools', result.project)
+    assert latest_pip_output.returncode == 0
+    pip_output = run('env/bin/pip3 install --editable .[dev]', result.project)
```

Comment:
If these tests are supposed to mimick how a user would install the package, probably `--editable` can go

```suggestion
    pip_output = run('env/bin/pip3 install .[dev]', result.project)
```

Reply:
I am mimicking a developer of the project . He/she would need editable flag.

Reply:
ok